File: solutions/go/hamming/1/hamming.go
package hamming

import "errors"

func Distance(a, b string) (int, error) {
	if len(a)!=len(b){
        return 0, errors.New("Strands must be equal length")
    }
    d := 0
    for i:=0;i<len(a);i++{
        if a[i] != b[i]{
            d=d+1
        }
    }
    return d, nil
}

File: solutions/python/hamming/1/hamming.py
def distance(strand_a, strand_b):
    if len(strand_a) != len(strand_b):
        raise ValueError("Strands must be of equal length.")
    
    differences = 0
    
    for a, b in zip(strand_a, strand_b):
        if a != b:
            differences += 1
    
    return differences
    